cves/kernel/CVE-2019-12378.yml

```diff
@@ -389,37 +388,39 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: true
+    note: |
+      An attacker would not need any privilege in order to exploit the bug, just needs local access.
```

Comment:
no need for a second needs after the comma.

Reply:
updated